src/ape_geth/provider.py

```diff
@@ -251,8 +248,8 @@ def is_likely_poa() -> bool:
                 else "Error getting chain id."
             )
 
-        # If network is rinkeby, goerli, or kovan (PoA test-nets)
-        if chain_id in (4, 5, 42) or is_likely_poa():
+        # If network is goerli or a custom PoA testnet
```

Comment:
Wait - Goerli is Proof of Stake now, so this should not be needed, right? @fubuloubu

Reply:
Yes! This should be true